packages/transactions/src/coders/receipt.ts

```diff
@@ -682,14 +682,22 @@ export class ReceiptMessageOutCoder extends Coder<ReceiptMessageOut, ReceiptMess
     value.data.forEach((d) => dataInNumbers.push(d));
     const parts: Uint8Array[] = [];
 
+    // encode data padded to nearest word (8 bytes)
```

Comment:
I think here you can use the `ByteArrayCoder `. Like this? https://github.com/FuelLabs/fuels-ts/blob/db8cc6b49616199368463ecd69aae6b3ca0b65d0/packages/transactions/src/coders/input.ts#L73

Reply:
Yep, thanks!